utils/getPrimaryColorInfo.ts

```diff
@@ -0,0 +1,28 @@
+import { Property } from '@stitches/react/types/css';
+import { PrimaryColor } from '../stitches.config';
+import { ColorMap } from '../colors';
+
+type ColorScale = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+
+type PickScale = (scale: ColorScale, options?: { alpha: boolean }) => Property.Color;
+
+export type ColorInfo = {
+  name: PrimaryColor;
+  token: Property.Color;
+  value: string;
+  helpers: {
+    pickScale: PickScale;
+  };
+};
+
+const getPrimaryColorInfo = (primaryColor: PrimaryColor, colors: ColorMap): ColorInfo => ({
+  name: primaryColor,
+  value: colors[`${primaryColor}9`],
+  token: `$${primaryColor}9`,
+  helpers: {
+    pickScale: (scale, options = { alpha: false }) =>
```

Comment:
:information_source: This anonymous function is not pure. 
I think it may make sense out of the `getPrimaryColorInfo` scope, so I'd rather make it a pure function that you reuse internally.

```
const pickScale = (primaryColor, scale, options) => ...
```